src/widget/form/profileform.cpp

```diff
@@ -247,10 +247,14 @@ void ProfileForm::onRenameClicked()
     Nexus& nexus = Nexus::getInstance();
     QString cur = nexus.getProfile()->getName();
     QString title = tr("Rename \"%1\"", "renaming a profile").arg(cur);
+
     do
     {
-        QString name = QInputDialog::getText(this, title, title+":");
-        if (name.isEmpty()) break;
+        QString name = QInputDialog::getText(0, title, title + ":");
```

Comment:
Removing the parent is no fix and raises another issue - dialog is mispositioned.

This issue is a bit less trivial, than it initially seems. It comes down to `QStyle` used in QTox (see file src/widget/style.h). This is set somewhere as application style and cannot be removed easily. I'm searching a way to tell the QTox style: "Don't apply to this widget." Will take some time though, sorry :(